stories/topics/proc-aap-aws-backup-and-recovery.adoc

```diff
@@ -7,113 +7,121 @@ Follow the steps below to ensure a smooth backup and recovery process.
-. Click btn:[Next] to go to the next step
-. Enter a stack name and select an AWSKeyPair to use for the deployment.
-. Click btn:[Next] to go to the *Configure stack options* page.
-. Optional: You can configure stack options and click btn:[Next] to review the stack
-. Click btn:[Submit] to create your new deployment using an RDS Snapshot and restored EFS.
-
-
-You can now log in succesfully to {PlatformName} {ControllerName} and {HubName} using your old deployment credentials. 
+. Gather the following input variables to create an `extra_vars.yaml` file.
+.. aws_foundation_stack_name: The name of your existing deployment
+.. aws_restored_stack_name: The name of your new deployment
+.. aws_region: The region the stack was deployed in
+.. aws_backup_vault_name: The name of the backup vault your EFS backup is stored in
+.. aws_rds_db_snapshot_arn: The ARN of the RDS snapshot you want to use for restore. Can be found from the backup playbook output as *rds_db_snapshot_arn*. 
+[NOTE]
+If you have a specific RDS snapshot you want to use, you will have to manually find its ARN in the AWS console. 
+.. aws_backup_restore_point_arn: The ARN of the EFS recovery point you want to use for restore. Can be found from the backup playbook output as *RecoveryPointArn*. 
+[NOTE] 
+If you have automatic backups set up and you have a specific EFS recovery point you want to use, you will have to manually find its ARN in the AWS console. 
+.. aws_backup_iam_role_arn: AWS IAM Role that has permissions to perform backup operations
+.. aws_s3_bucket: AWS S3 bucket name. Used to upload cloudformation template
+.. aws_efs_physical_id: The physical Id of the EFS from the original deployment.  Looks something like `fs-06837574544929090`  
+.. aws_cf_keypair_name: The keypair to pass as a parameter when creating the deployment.
+[NOTE]
+The keypair used must exist in the aws region you are restoring to
+
+
+. Pull the docker image for operational container having the tag with the same version as the foundation deployment.
++
+[source,bash]
+----
+$ export IMAGE=<image-path still needs to be determined>
+$ docker pull $IMAGE
+----
++
+
+[NOTE]
+=====  
+Operational image tag should match with the version of your foundation deployment. For example, if your foundation deployment version is 2.2.20230215, pull the operational image with tag 2.2.20230215.
+=====
+. Running the restore playbook as a container.
+
+.. Replace `<absolute path of extra vars dir>` with the path to your directory where your `extra_vars.yaml` file exists which you have created in step 1 of the backup process and replace `<Your AWS Access Key ID>` and `<Your AWS Secret Access Key>` with the values for the AWS account you want to run the backup with. For more information on finding your AWS account credentials refer to https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-about
```

Comment:
For more information on finding your AWS account credentials, see link:https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-about[About the access keys] 

whatever the document is called in the []